.github/workflows/go.yml

```diff
@@ -0,0 +1,25 @@
+name: Go
+on: [push]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Build
+      run: |
+        export PATH=$HOME/go/bin:$PATH
+        go get -u golang.org/x/lint/golint
+        go fmt ./...
```

Comment:
This can leave you git working dir in dirty state. Problematic when you want to further release it. Also I think a pre-commit hook is more suitable for that. WDYT?

Reply:
The idea of this first PR is to get Github Actions added. I'll make a more complete build process as I can continue working on this code.